app/models/edition.rb

```diff
@@ -96,6 +96,7 @@ class Edition < ApplicationRecord
   scope :future_scheduled_editions,     -> { scheduled.where(Edition.arel_table[:scheduled_publication].gteq(Time.zone.now)) }
 
   # @!group Callbacks
+  before_create :set_auth_bypass_id
```

Comment:
I wonder if it would be possible to add a test for this?  Maybe in `EditionTest` you could create a new edition and check the `auth_bypass_id` field is not nil.

Reply:
Thnak you @brucebolt We are going to add not NULL restraint as a separate PR (see conclusions here https://github.com/alphagov/whitehall/pull/6514/files#r857406916), and I was thinking to add a test then?

Reply:
I guess you could add a test now to ensure that newly created editions have a `auth_bypass_id`.  You could then extend this to check for the NULL constraint in the next PR?